guides/doc-Upgrading_and_Updating/topics/upgrading_external_database.adoc

```diff
@@ -0,0 +1,25 @@
+[id="Upgrading_External_Database_{context}"]
+= Upgrading External Database
+
+You can upgrade an external database from {EL} 7 to {EL} 8 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+
+.Prerequisites
+. Create a backup.
++
+* On a virtual machine, take a snapshot.
+* On a physical machine, create a backup.
+. Create a new {EL} 8 based host for PostgreSQL server that follows the external database on {EL} 8 documentation.
+
+.Procedure
+. If you keep the same hostname on the new PostgreSQL server, you need to update the DNS preventing the need to do updates on {Project} itself.
+. If the new PostgreSQL server has a new host name or a different CNAME, update the reference to the external database on {Project}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} \
+  --foreman-db-host newpostgres.example.com \
+  --katello-candlepin-db-host newpostgres.example.com \
+  --foreman-proxy-content-pulpcore-postgresql-host newpostgres.example.com \
+----
+. Copy the backup database to the new host.
+. Restore the backup database on the new host.
```

Comment:
Would it make sense to give explicit instructions for this? Like how to use `pg_dump` and `pg_restore`.

Reply:
@ehelms your thoughts?

Reply:
The stance has been that if a user under takes running an external database server they are expected to take care of doing a backup and restore and the details of those procedures is out of scope of our documentation. Our intent with including an OS upgrade guide is more to give guidance the order of operations and general steps a user should take but the details are up to them based on their database.